flent/metadata.py

```diff
@@ -514,3 +514,27 @@ def get_module_versions():
             module_versions[m] = v
 
     return module_versions
+
+def get_wifi_data(iface):
```

Comment:
You'll also need to add appropriate calls to this function in record_metadata(); I'd suggest putting them along with the calls to get_ip_addrs() and get_gateways()

Reply:
I didn't get it

Reply:
You added a new function. But it doesn't do anything until you add some code that calls that function and does something with the result. Look at the how get_ip_addrs() is being called in record_metadata() for an example...